### DISCUSSION TITLE
Got feedback? Chat with the GitHub Dependabot team!

### DISCUSSION THREAD

**Discussion by carlincherry** [dependabot/dependabot-core#9345] [Announcements]
📢  Would you like to give your input directly to the GitHub Dependabot team? We are always looking for users to interview to learn more about how you use Dependabot and what you wish was better.

🥇 If you'd like an update on a specific issue or request, feel free to email me: carlincherry (at) github.com.

🧠 If you have 30 mins available in North American timezones, and would like to speak synchronously, you can use this link to book a meeting directly with me, Dependabot's PM: https://calendar.app.google/sfybZVNtzPPjpdCq8

🚀 Looking forward to connecting with you!

**Comment by James-E-A** [comment]
For people who would like to use Dependabot but currently aren't users, is this Discussions forum the correct place for feedback?

https://www.github.com/dependabot/dependabot-core/discussions/9327

**Reply by carlincherry** [reply]
Hi @JamesTheAwesomeDude! The best place to go for feedback and/or Feature Requests like the one you have in the discussion post is to file an [Issue](https://github.com/dependabot/dependabot-core/issues) in `dependabot-core` rather than a discussion post. Thanks for your feedback!

**Comment by jsoref** [comment]
Fwiw, there's a [Give feedback](https://gh.io/dependabot-feedback) link that's present at the top of `github.com/{org}/{repo}/security/dependabot` which yields an error from Google Calendar.

---

At some point I'll try to schedule a meeting.

**Reply by carlincherry** [reply]
You are absolutely right -- thank you for sharing. We'll get that bug fixed!

**Reply by carlincherry** [reply]
You can schedule a feedback session with me [here](https://calendar.app.google/2BX9orkqcGon9grY9) for a live meeting, or send me an email at carlincherry@github.com too. Always appreciate your time and feedback!

**Comment by shikhahere** [comment]
Hello @carlincherry i wanted to discuss about issue [#13012](https://github.com/dependabot/dependabot-core/issues/13012) and [#16206] .
specifically on how to make the experimental feature available for teams. And how i can help further contribute.

i scheduled a call for today in your calendar.  Are feedback sessions still an option?
